netutils/acl.py

```diff
@@ -513,9 +570,9 @@ def match(self, rule: ACLRule) -> str:
             The response from the rule that matched, or `deny` by default.
         """
         for item in self.rules:
-            if item.match(self.class_obj(rule)):
-                return str(item.action)
-        return str(item.deny)  # pylint: disable=undefined-loop-variable
+            if item.match(rule):  # mzb: bugfix
+                return True  # mzb: change to bool
+        return False  # pylint: disable=undefined-loop-variable
```

Comment:
I had this as string as most vendors have more than just permit or deny, this is Palo's

```
allow
deny
drop
reset client
reset server
reset client and server
```

Reply:
IMO `match` method should return only True/False if the rule is/not matched by the check. This should only check if A contains B.

Reply:
I think it is reasonable to have different levels.

Reply:
I think, I see this in a little different way :
```
def match_action(action, match_action):
    deny_actions = ['deny', 'drop', 'reset client', 'reset server', ...]
    if action in deny_actions and match_action in deny_actions:
        return True
    else:
        return False
```

I would really like to avoid putting opinionated logic into generic `match` resolution while we provide easily extensible framework